src/process_viewer.rs

```diff
@@ -480,8 +480,9 @@ fn build_ui(application: &gtk::Application) {
         // To make "run" and "cancel" button take all spaces.
         if let Some(run) = dialog.get_widget_for_response(gtk::ResponseType::Other(0)) {
             if let Some(parent) = run.get_parent() {
-                let parent = parent.downcast::<gtk::ButtonBox>().unwrap();
```

Comment:
Weird, it should never be an issue... Can you use a `match` instead and print a message in case of error please?

Reply:
Sure. Hopefully the error message is ok for you.

Reply:
Yup, thanks!